examples/shader/post_processing.rs

```diff
@@ -42,11 +42,43 @@ fn setup(
 ) {
     let window = windows.primary_mut();
     let size = Extent3d {
-        width: window.physical_width(),
-        height: window.physical_height(),
+        width: window.width() as u32,
+        height: window.height() as u32,
         ..default()
     };
 
+    commands
```

Comment:
The point of this is example is to minimally illustrate a post processing effect. This UI serves no purpose other than to prove that a bug has been resolved. It introduces additional complexity that distracts from the purpose, so I think it should be removed.